src/main/kotlin/com/sourcegraph/cody/CodyToolWindowContent.kt

```diff
@@ -40,29 +52,53 @@ class CodyToolWindowContent(project: Project) {
     val codyAuthenticationManager = CodyAuthenticationManager.getInstance()
     if (codyAuthenticationManager.hasNoActiveAccount() ||
         codyAuthenticationManager.showInvalidAccessTokenError()) {
-      allContentLayout.show(allContentPanel, SIGN_IN_PANEL)
+      cardLayout.show(cardPanel, SIGN_IN_PANEL)
+      showView(cardPanel)
       return
     }
     val activeAccount = codyAuthenticationManager.account
     if (!CodyApplicationSettings.instance.isOnboardingGuidanceDismissed) {
       val displayName = activeAccount?.let(CodyAccount::displayName)
-      allContentPanel.getComponent(ONBOARDING_PANEL_INDEX)?.let {
+      cardPanel.getComponent(ONBOARDING_PANEL_INDEX)?.let {
         (it as CodyOnboardingGuidancePanel).updateDisplayName(displayName)
       }
-      allContentLayout.show(allContentPanel, ONBOARDING_PANEL)
+      cardLayout.show(cardPanel, ONBOARDING_PANEL)
+      showView(cardPanel)
       return
     }
-    allContentLayout.show(allContentPanel, MAIN_PANEL)
+    cardLayout.show(cardPanel, LOADING_PANEL)
+    showView(webviewPanel ?: cardPanel)
```

Comment:
That looks like potential source of hidden issues.
If `webviewPanel` will be null when we expect it to be not, and we switch to cardPanel instead, won't it be stuck forever in non-webview mode?

Reply:
This is just like `MAIN_PANEL` used to be: either it is the webview, or it is the loading indicator.